### PR TITLE
[RELEASE-6.2.3] Remove SNAPSHOTs

### PR DESCRIPTION
**Pull request by CuriousVini**
This is an automated PR to remove -SNAPSHOT from artifact versions to prepare for release.

### CODE REVIEW

File: cdap-docs/user-guide/source/pipelines/plugins/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.hydrator</groupId>
   <artifactId>hydrator-plugins</artifactId>
-  <version>1.7.1-SNAPSHOT</version>
```
Comment (supriyapatil-google) [inline cdap-docs/user-guide/source/pipelines/plugins/pom.xml]: Is this expected?

**Review by supriyapatil-google — APPROVED** [review]
Thanks!

File: cdap-docs/user-guide/source/pipelines/plugins/pom.xml (hunk 1 above)
Comment (CuriousVini) [reply cdap-docs/user-guide/source/pipelines/plugins/pom.xml]: Thanks for catching that! Its not intended, fixed

**Comment by CuriousVini** [comment]
Reference PR: https://github.com/cdapio/cdap/pull/12780
squashing and merging

**Pull request merged**